<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVariableDefine</class>
 <widget class="QDialog" name="DialogVariableDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QGroupBox{
	background-color: rgba(16, 57, 70, 1);
	border: 0px solid rgb(111, 255, 255);
	margin-top:3.3ex;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title{
	subcontrol-origin: margin;
	subcontrol-position: top center;
	background-color: rgba(19, 67, 79, 1);
	padding: 7 10000px;
}

QLineEdit{
	background-color: rgba(255, 255, 255,0.2);
	border-width:0;
	border-style:outset;
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QLabel{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton{
	color: rgba(102, 249, 247, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QRadioButton::indicator{
	border-color: rgba(255, 255, 255,0);
	background:transparent;
}


QRadioButton::indicator:checked{
	border-image:url(:/widget/images/green.png)
}

QRadioButton::indicator:unchecked{
	border-image:url(:/widget/images/unchecked.png)
}

QPushButton{
	border: 2px solid rgba(61, 255, 255,0.4);
	background-color: rgba(12, 34, 45, 0.6);
	font: 12pt &quot;微软雅黑&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
}

QPushButton::disabled{
	border: 0px;
	color: rgba(102, 249, 247, 0.3);
	text-align：center;
}

QPushButton::pressed{
	border: 2px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 1);
	font: 12pt &quot;微软雅黑&quot;;
}

QPushButton::hover{
	border: 2.5px solid rgba(61, 255, 255,0.8);
	background-color: rgba(12, 34, 45, 0.6);
	font: 75 12pt &quot;微软雅黑&quot;;
}

QDoubleSpinBox{
	border: 0px solid rgba(61, 255, 255,0.4);
	background-color: rgba(255, 255, 255, 0.2);
	font: 12pt &quot;Times New Roman&quot;;
	color: rgba(102, 249, 247, 1);
	text-align：center;
	padding-left:10px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1500</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox
{
background-color: rgb(14, 118, 141);
border-radius: 15px; 
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn_OK">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>410</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancel">
    <property name="geometry">
     <rect>
      <x>417</x>
      <y>410</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_warning">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>40</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>40</y>
      <width>931</width>
      <height>350</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>变量名称</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>信号类型</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>变量类型</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>变量单位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>变量位置</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>报警内容</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>规则库自动定位故障点</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>模糊组故障点</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBox_multipos">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>NULL</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="btn_AddNewMultiPos">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string> 增加</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="btn_ModMultiPos">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="btn_DelMultiPos">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_SignalType">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>中间信号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>故障信号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>报警信号</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_VariableType">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>BOOL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ENUM</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DOUBLE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>INT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_unit">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>NULL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>°C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MPa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>m³/s</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>V</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="comboBox_signalpos">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>中心控制箱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>机旁控制箱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>备用泵电机启动箱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>提升泵电机启动箱</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>液压系统</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_note">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="4">
      <widget class="QLineEdit" name="lineEdit_detail">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogVariableDefine</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
